### ISSUE TITLE
Please add an "Add Friend" button

### ISSUE THREAD

**Issue by juanchodepisa** [online-go/online-go.com#346]
### Summary

Upon clicking on someone's username there are various possibilities, including challenging, PM, and even block. Yet there's no option to add them as a friend.

It is unclear to me whether this was a purposeful design choice. However, it strikes me as a bit odd that there are arguably two actions to drive a wedges between two users (report and block) and none to bond them together.

In short, I find it is easier to block another user than to befriend them, which IMHO is a bit sad, specially considering that this server is based on a community.

So, please consider adding an "Add Friend" button.

**Comment by Spoekenkieker** [comment]
I like that a LOT.

**Comment by anoek** [comment]
Agreed
